src/rest_api.py

```diff
@@ -1,43 +1,28 @@
 """Implementation of the REST API for the backbone service."""
 
-import os
-import logging
 import flask
+import logging
+import os
 import time
+
 from f8a_worker.setup_celery import init_selinon
 from flask import Flask, request
 from flask_cors import CORS
 from raven.contrib.flask import Sentry
 
 from src.recommender import RecommendationTask as RecommendationTaskV1
 from src.stack_aggregator import StackAggregator as StackAggregatorV1
+from src.utils import push_data, total_time_elapsed
 from src.v2.recommender import RecommendationTask as RecommendationTaskV2
 from src.v2.stack_aggregator import StackAggregator as StackAggregatorV2
-from src.utils import push_data, total_time_elapsed, get_time_delta
-
-
-def setup_logging(flask_app):
```

Comment:
Refer entrypoint.sh changes to understand why `setup_logging` has been removed

Reply:
```
[2021-01-06 13:46:04,266] INFO in /src/v2/stack_aggregator.py:300: hello took 4.93 secs for get_license_analysis_for_stack()
```
This is the log print from default format, which matches the required behaviour.